experimental/dds/tree2/src/test/forestTestSuite.ts

```diff
@@ -153,22 +153,55 @@ export function testForest(config: ForestTestConfiguration): void {
 
 		it("moving a cursor to the root of an empty forest fails", () => {
 			const forest = factory(new InMemoryStoredSchemaRepository(defaultSchemaPolicy));
-			const dependent = new MockDependent("dependent");
-			recordDependency(dependent, forest);
 			const cursor = forest.allocateCursor();
 			moveToDetachedField(forest, cursor);
 			assert.equal(cursor.firstNode(), false);
 		});
 
-		it("anchors creation and use", () => {
+		it("tryMoveCursorToField", () => {
```

Comment:
Should this be `tryMoveCursorToNode`?

Reply:
Fixed. Also added tests for tryMoveCursorToField